fenix/app/src/main/java/org/mozilla/fenix/theme/FirefoxTheme.kt

```diff
@@ -167,6 +168,7 @@ private val lightColorPalette = FirefoxColors(
     layer1 = PhotonColors.LightGrey10,
     layer2 = PhotonColors.White,
     layer3 = PhotonColors.LightGrey20,
+    layerSearch = PhotonColors.LightGrey30,
```

Comment:
Can you move this after `layerInfo`? Ideally, the order of these tokens follow what we see in https://www.figma.com/file/pEyGeE4KV5ytYHeXMfLcEr/Mobile-Styles?type=design&node-id=5276-17056&mode=design&t=xcAvTpSfMtjTh0HY-0. However, we are overdue on updating this to align with the latest changes in our color token stickersheet. I will file a follow up to update things.

The reasoning behind ordering things to be the same exact order of the stickersheet is to make it easier to maintain the list of colors. It is easiest for us when we can just look at the figma table and everything is in order when we are just making sure that 1) we have the newly added token, 2) can verify the color and token matches without jumping around the code.

Reply:
We also don't have to move this now. Just understanding that we will want to reorder things in the future is also good enough.

Reply:
Oh thanks for this piece of information!